src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProvider.Searcher.cs

```diff
@@ -124,27 +128,39 @@ private class Searcher
 #pragma warning disable CS0618 // MatchKind is obsolete
             private void ReportMatchResult(Project project, INavigateToSearchResult result)
             {
+                ImmutableArray<Span> arrayTextSpan = ImmutableArray<Span>.Empty;
+                foreach (var s in result.NameMatchSpans)
+                {
+                    arrayTextSpan.Add(new Span(s.Start, s.Length));
```

Comment:
this does nothing (since arrayTextSpan is immutable).  I think you want to do:

```c#
var builder = ArrayBuilder<Span>.GetInstance()
... add the items.
var arrayTextSpan = builder.ToImmutableAndfree();
```

Reply:
Ahhh... I was wondering about that.  Thanks